basyx.aasrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/http/TestAasRepositoryHTTP.java

```diff
@@ -187,6 +187,51 @@ public void removeNonExistingSubmodelReference() throws FileNotFoundException, I
 		assertEquals(404, deleteResponse.getCode());
 	}
 
+	@Test
+	public void getAssetInformationByIdentifier() throws FileNotFoundException, IOException, ParseException {
+		createDummyAasOnServer();
+		String url = getSpecificAasAccessURL(dummyAasId) + "/asset-information";
```

Comment:
Please extract the _getSpecificAasAccessURL(dummyAasId) + "/asset-information";_ into its own function and utilize it in the other test cases